<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator</class>
 <widget class="QDialog" name="calculator">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;calculator&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 34, 34);</string>
  </property>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(192, 193, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_point">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 127);
font: 700 12pt &quot;Segoe UI&quot;;


</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_multiply">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 127);
font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>ร</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sub">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(170, 85, 127);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sum">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_equal">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 10, 34);
font: 700 12pt &quot;Segoe UI&quot;;

</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
background-color: rgb(198, 66, 0);
</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_symmetry">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(255, 85, 0);
</string>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_percent">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_division">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 0, 127);
font: 700 12pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>รท</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(0, 85, 127);

</string>
   </property>
   <property name="text">
    <string>Sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cos">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);
font: 700 9pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tan">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
font: 700 9pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cot">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>Cot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_log">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sqrt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);
font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Sqrt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(106, 106, 106);
font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(0, 85, 0);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
